conf/application.conf

```diff
@@ -144,10 +144,14 @@ mongodb {
 }
 
 urls {
+  businessAccountAuthority = "http://localhost:9020"
+
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:9000/business-account/corporation-tax"
 
-  makeAPayment = "http://localhost:9050/pay-online/corporation-tax/make-a-payment?mode=bta"
+  payApiBase = "http://localhost:9057/pay-api"
```

Comment:
@alex-newby-hmrc should this not be using port 8080 (bta-stubs)? We've decided to use bta-stubs for vat-frontend which I think is the correct thing to do.

Reply:
@lmugnai 
My understanding of the situation is this:
The config key `payApiBase` is used for two purposes:
a) starting a payment journey
b) retrieving historic payments via the RTP API.
This issue is exactly the same in `vat-frontend`. I spoke to Seymour about how you guys wanted to have this point to the stubs so you could control the data returned by the `b)` case, which makes sense to me. However, if you point it at the stubs you will break the code in the `a)` case because, as far as I know, your stubs don't have the capacity to start a payment journey.
The solution that Seymour and I came to for `vat-frontend` was that we would have to split the config key in two: one for starting a payment journey and one for historic payments. Then you can point the latter at the stubs, while the former still points at the real `pay-api`. I assume that would be a good idea here as well.
If you agree with this config-key-split solution, and you need me to implemented it, I would request that we merge this as is, because I'm quite busy/pressed for time at the moment. I am worried about this branch getting stale, and if there are any bugs I am happy to work on those so we can get it out the door ASAP.